test/functional/test-amp-video.js

```diff
@@ -273,6 +275,19 @@ describe('amp-video', () => {
     });
   });
 
+  it('should dispatch VideoEvents.USER_TAP event when tapped', () => {
+    return getVideo({
+      src: 'video.mp4',
+      width: 160,
+      height: 90,
+    }).then(v => {
+      const video = v.querySelector('video');
+      const p = listenOncePromise(v, VideoEvents.USER_TAP);
+      video.dispatchEvent(new Event('click'));
+      return p;
```

Comment:
Yeah, gets the Mocha's default 2 second timeout and fails the test (have tested it)